src/main/java/core/basesyntax/service/impl/ParseServiceImpl.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ParserService;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ParseServiceImpl implements ParserService {
```

Comment:
```suggestion
public class ScvParseServiceImpl implements ParserService {
```
In fact you are parsing data from SCV file in this service